### PR TITLE
build: package updates!

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[![PR App][icn]][demo] |
:-------------------:|

## 🧰 Changes

Updates a bunch of packages.

Holding back `jest` till `jest-image-snapshot` can update it's deps.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: .npm-upgrade.json [hunk 1]
```diff
@@ -79,6 +75,14 @@
     "react-dom": {
       "versions": "^18",
       "reason": "Waiting on react"
+    },
+    "jest": {
```
Comment (erunion) [inline .npm-upgrade.json:79]: Does Dependabot pick up this file?

File: __tests__/components/HTMLBlock.test.jsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ const { render, screen } = require('@testing-library/react');
 const React = require('react');
 const { renderToString } = require('react-dom/server');
 
+// eslint-disable-next-line import/order
```
Comment (erunion) [inline __tests__/components/HTMLBlock.test.jsx:5]: These imports need to be loaded in this order?

**Review by erunion — APPROVED** [review]

File: .npm-upgrade.json (hunk 1 above)
Comment (kellyjosephprice) [reply .npm-upgrade.json:79]: Not at all, but that'd be really cool.

Comment (erunion) [reply .npm-upgrade.json:79]: Might want to replicate these into the dependabot config so it doesn't try to update these.

File: __tests__/components/HTMLBlock.test.jsx (hunk 2 above)
Comment (kellyjosephprice) [reply __tests__/components/HTMLBlock.test.jsx:5]: Let me double check that, but I think eslint was getting confused.

Comment (kellyjosephprice) [reply __tests__/components/HTMLBlock.test.jsx]: Ok, converted to esm and cleaned up to remove the exception.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.53.1</b>](/readmeio/markdown/releases/tag/v6.53.1)
<br>

**Pull request merged**